packages/ra-ui-materialui/src/layout/UserMenu.tsx

```diff
@@ -31,7 +34,7 @@ const UserMenu = props => {
     const handleClose = () => setAnchorEl(null);
 
     return (
-        <div className={classes.user}>
```

Comment:
this is a breaking change. Can you please revert it?

Reply:
Done!